### ISSUE TITLE
AttributeError: 'Cells' object has no attribute 'pop' Error

### ISSUE THREAD

**Issue by rajeshkz** [pytest-dev/pytest-html#606]
I'm getting an error "AttributeError: 'Cells' object has no attribute 'pop'" with the version of  4.0.0rc0. The error is coming from when executing the code mentioned at https://pytest-html.readthedocs.io/en/latest/user_guide.html#modifying-the-results-table 
The issue gets resolved when the version of pytest-html was downgraded to 3.2.0 
pytest Version: 7.2.2

**Comment by BeyondEvil** [comment]
Thanks for reporting!

This has been fixed but not yet released. It will be part of the RC1 release.

Note that the fix is to remove the call to `pop`.

**Comment by rajeshkz** [comment]
Thanks for the update. Would this document be updated case the call to pop is removed? 
https://pytest-html.readthedocs.io/en/latest/user_guide.html#modifying-the-results-table

**Comment by BeyondEvil** [comment]
> Thanks for the update. Would this document be updated case the call to pop is removed? https://pytest-html.readthedocs.io/en/latest/user_guide.html#modifying-the-results-table

Yes, it's also on the branch: https://github.com/pytest-dev/pytest-html/blob/next-gen/docs/user_guide.rst#modifying-the-results-table

**Comment by BeyondEvil** [comment]
Please try `4.0.0rc1` 🙇
